decidim-initiatives/app/views/decidim/initiatives/admin/initiatives/show.html.erb

```diff
@@ -99,6 +99,9 @@
 <h2 class="print-section-title">Sol·liciten que sigui admesa a tràmit la Iniciativa Ciutadana que porta el títol de:</h2>
 <%= translated_attribute(current_initiative.title) %>
 
+<h2 class="print-section-title">Tipus de la Iniciativa:</h2>
```

Comment:
I just realised this file is all in catalan 😳

@decidim/product can you provide the translations?

Reply:
This feature was made for Barcelona's council. We, as members of Decidim crew were not in favour of adding it. In fact the plan was to remove it as soon as possible. It exists only because Barcelona's council requires from users to physically present the initiative in a office.

On the other hand as far as I know there are plans to make a deep refactor on this feature in the near future.

From this perspective, and since it was accepted without translations and the only reason for this PR was to fix a critical but unexistent feature, I think we should ask to @xabier and @arnaumonty if it has sense to invest time on it at this moment.